docs/how-to/work-with-pinning-services.md

```diff
@@ -3,8 +3,11 @@ title: Work with pinning services
 description: Learn how to use or create remote pinning services with IPFS, the InterPlanetary File System.
 ---
 
+
 # Work with remote pinning services
 
+[[toc]]
```

Comment:
Can we please avoid these? It's duplication for people who are viewing this on docs.ipfs.io.

Reply:
In this specific case toc here is useful, because it provides links to GUI and CLI sections, while the outline in the left menu does not:

>  ![2021-06-25--00-02-32](https://user-images.githubusercontent.com/157609/123338193-aeaf0780-d548-11eb-945e-8e697063785e.png)

Makes it way easier for people to jump to specific guide.